setup.py

```diff
@@ -66,6 +66,7 @@
                 'pyroute2.remote'],
       scripts=['./cli/ss2',
                './cli/pyroute2-cli'],
+      extras_require={'ss2': ['psutil>=5.0']},
```

Comment:
```suggestion
      extras_require={'ss2': ['psutil~=5.0']},
```
you will still want an upperbound, as 6.0 may break your use cases (~=5.0 -> >=5,<6)